src/py42/services/storage/_auth.py

```diff
@@ -0,0 +1,64 @@
+from py42.services._auth import C42RenewableAuth
+
+
+class StorageTmpAuth(C42RenewableAuth):
+    def __init__(self):
+        super(StorageTmpAuth, self).__init__()
+        self._storage_url = None
+
+    def get_storage_url(self):
+        self.get_credentials()
+        return self._server_url
+
+    def get_tmp_auth(self):
+        raise NotImplementedError()
+
+    def _get_credentials(self):
+        login_info = self.get_tmp_auth()
+        login_token = login_info[u"loginToken"]
+        self._server_url = login_info[u"serverUrl"]
+        return u"{} {}".format(u"login_token", login_token)
```

Comment:
No need to include hardcoded strings in a format statement, you can just put it in the original string